tests/general/SpaceConsistencyBearTest.py

```diff
@@ -70,3 +77,20 @@ def test_enforce_newline_at_eof(self):
                                "    print('funny')\n",
                                "    print('the result is not funny...')"],
                               force_linebreaks=False)
+
+    def test_leading_blanklines(self):
+        self.section.append(Setting('use_spaces', 'true'))
+        self.section.append(Setting('allow_trailing_whitespace', 'false'))
+        self.section.append(Setting('enforce_newline_at_EOF', 'true'))
+        self.section.append(Setting('allow_leading_blanklines', 'false'))
+
+        self.check_invalidity(self.uut,
+                              ['\n',
+                               '  \n',
```

Comment:
From this test case, I cannot see whether the invalidity is due to a blank line (first line), or trailing white spaces (second line).

Reply:
You should also verify the error message.